modules/programs/waybar.nix

```
@@ -161,7 +161,7 @@ let
       };
     };
 in {
-  meta.maintainers = [ hm.maintainers.berbiche ];
+  meta.maintainers = with pkgs.stdenv.lib.maintainers; [ berbiche ];
```

```suggestion
  meta.maintainers = with lib.maintainers; [ berbiche ];
```

`lib` is already in scope, so why use `pkgs.stdenv.lib`? :P

---

Good catch!